### PR TITLE
Adds thoughts on sprint demos

### PR DESCRIPTION
**Pull request by iteles**
Initial thoughts on sprint demos as per #5 with a TODO for the stuff I haven't had a chance to write yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+Some suggestions:
++ Prep beforehand
+  + Review your backlog and take note of all the work you have done in the sprint
+  + Walk through the proposed demo, paying close attention to:
+    + Picking up on any small niggles the client may notice
+      + Can you preempt the client bringing it up by proposing a solution as you're doing the demo?
+      + Are there things you're actually still not 100% sure are working the way the client intends? Ask the questions during the demo so they know you're looking out for their needs
+    + All of the work that has been done so that you can point it out during the demo (especially if additional work has been done that wasn't originally expected) 
++ Have all changes merged into `master` at least half a day before the demo
+  + The rushed changes you make in the last half a day are usually the changes that break everything and cause stress during the demo
```
Comment (nelsonic) [inline README.md]: I would not use the word `"break"` here... but I _understand_ what you're trying to say...

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+  + Your last day/half a day should be for polish and bug fixing from the merges, not for functionality
++ Breathe - this can be nerve wracking and taking some deep breaths will help slow you down!
++ Start the demo by providing some context such as what the aim of the sprint was and what you'll be demonstrating
+  + Be calm and deliberate
+  + The purpose here is two-fold; it sets up the demo but more importantly it allows attendees time to gently have a little time to switch their attention away from their previous tasks (or meeting they have just come from), slow down and focus on the task at hand
```
Comment (nelsonic) [inline README.md]: Always work from a list of steps ("script") in your demo to ensure you don't forget anything.

**Comment by nelsonic** [comment]
@iteles looks good. 👍 
left a couple of comments. 💬 
we can enhance it in the next iteration. ♻️

**Comment by nelsonic** [comment]
also, how come _you_ aren't ⭐ -ing this repo...? 😮

**Comment by iteles** [comment]
Agreed, there's still a way to go on this repo!

**Pull request merged**